### PR TITLE
Dev 1341

### PR DESCRIPTION
**Pull request by dvanbeek**
Adding Frequently Asked Questions page and page with instructions on how to complete a Data Access Request application. Other small changes specified in DEV-1341 were made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
-* [Accessing HMF Data through Google Cloud Platform](accessing-hmf-data-through-gcp.md): This document contains instruction on how to use GCP to download or process HMF data. 
-* [HMF Data Access Request Guide](data-access-request-guide.md): This document contains information regarding the files you receive when your Data Access Request is approved. How are the samples selected, what formats will you receive.
-* [HMF Data Access Request Methods](data-access-request-methods.md): This document contains the methods that were used to generate the data that is serviced through Data Access Requests.
+* [Instructions on completing your Data Access Request application](data-access-request-application): A walkthrough of the application that needs to be filled out when applying for Hartwig data.
+* [Getting a Google Cloud Platform Account](getting-a-Google-account.md): Instructions on how to set up a Google Account for accessing the Hartwig Medical Foundation data set.
+* [Accessing Hartwig Data through Google Cloud Platform](accessing-hartwig-data-through-gcp.md): Instruction on how to use GCP to download or process HMF data.
```
Comment (pauldwolfe) [inline README.md]: Instructions

File: README.md [hunk 2]
```diff
@@ -2,16 +2,18 @@
...
-* [HMF Data Access Request Guide](data-access-request-guide.md): This document contains information regarding the files you receive when your Data Access Request is approved. How are the samples selected, what formats will you receive.
-* [HMF Data Access Request Methods](data-access-request-methods.md): This document contains the methods that were used to generate the data that is serviced through Data Access Requests.
+* [Instructions on completing your Data Access Request application](data-access-request-application): A walkthrough of the application that needs to be filled out when applying for Hartwig data.
+* [Getting a Google Cloud Platform Account](getting-a-Google-account.md): Instructions on how to set up a Google Account for accessing the Hartwig Medical Foundation data set.
+* [Accessing Hartwig Data through Google Cloud Platform](accessing-hartwig-data-through-gcp.md): Instruction on how to use GCP to download or process HMF data. 
+* [Data Access Request Guide](data-access-request-guide.md): Information regarding the files you receive when your Data Access Request is approved. How are the samples selected, what formats will you receive.
```
Comment (pauldwolfe) [inline README.md]: Sentence fragment. " Data Access Request is approved: how are samples are selected; which formats will you receive." would fix.

File: data-access-request-faq.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+### Will I be billed for accessing your data?
+We don't personally bill you for accessing our data, but accessing the data through Google Cloud Platform does come with costs. Basically, there are two ways to approach the data:
+- **Download the data onto your own servers**. This will result in egress costs (data that is pulled over the network will be charged per GB, see [Google's Network Pricing page](https://cloud.google.com/compute/network-pricing) for more details.
+- **Analyse the data on GCP itself**. This will results in storage and compute costs. More information can be found on [Google's Cloud Storage pricing page](https://cloud.google.com/storage/pricing) and [Google's VM Instances pricing page](https://cloud.google.com/compute/vm-instance-pricing). If you analyse the data in the same region as it is stored, you won't be charged egress costs. 
+
+What option to choose depends on your use case and specific requirements. 
+
+### How much will I get billed for my download?
+A resource that might help you in estimating costs is the [Google Pricing calculator](https://cloud.google.com/products/calculator).
```
Comment (pauldwolfe) [inline data-access-request-faq.md]: I'm not sure if the calculator has network costs, good to link the network pricing page here.

**Review by pauldwolfe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dvanbeek) [reply README.md]:
```suggestion
* [Accessing Hartwig Data through Google Cloud Platform](accessing-hartwig-data-through-gcp.md): Instructions on how to use GCP to download or process HMF data. 
```

File: README.md (hunk 2 above)
Comment (dvanbeek) [reply README.md:17]:
```suggestion
* [Data Access Request Guide](data-access-request-guide.md): Information regarding the files you receive when your Data Access Request is approved: how samples are selected; which formats will you receive.
```

File: data-access-request-faq.md (hunk 3 above)
Comment (dvanbeek) [reply data-access-request-faq.md]:
```suggestion
A resource that might help you in estimating costs is the [Google Pricing calculator](https://cloud.google.com/products/calculator). Another factor that you need to include when estimating cost, is the [cost for egress](https://cloud.google.com/compute/network-pricing).
```

File: README.md (hunk 2 above)
Comment (dvanbeek) [reply README.md]:
```suggestion
* [Data Access Request Guide](data-access-request-guide.md): Information regarding the files you receive when your Data Access Request is approved: how are samples selected; which formats will you receive.
```

**Pull request merged**